README.md

```diff
@@ -1,3 +1,7 @@
+# Status
+
+Note: This repo is not being actively maintained by the authors as the FreeDV 2020 mode for which it was developed does not perform well on HF channels and is not widely used. You are welcome to use this repo as is, or fork it, but regretably we are unavailable to assist with Issues and Pull Requests.  It has been an interesting experiment which we have now completed.
```

Comment:
I think we should keep open the possibility of fixing any build problems on *supported* platforms (x86/ARM) and OSes (Linux/macOS generally and MinGW for Windows) while 2020/2020B are still user-selectable options in freedv-gui. Anything more substantial we don't need to support.

Reply:
Good points - I've tweaked the notice.